site/quickstart.md

```diff
@@ -237,10 +237,10 @@ e.	To confirm that the load balancer noticed the new Ingress and is successfully
 **NOTE**: Be sure to include the trailing forward slash on the URL, otherwise the command won't work.
 
 ```
-$ curl -v -H 'host: sample-domain1.org' http://your.server.com:30701/weblogic/
-* About to connect() to your.server.com port 30701 (#0)
+$ curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/
```

Comment:
I have not tested the port. Invited Lily to take a look at this.

Reply:
Just a bit of history. The port number, 30305, came from Lily (I believe) and has always been the port number in the curl command in the Quick Start guide. Recently, Monica gave me the port number to use for WLS Administration Console access, which is port 30701. I changed all the port numbers to 30701, then was informed by Monica that 30305 is the correct port number for the LB whereas 30701 is the correct port number for the admin console, hence the edit. I expect that Lily will confirm this change.

Reply:
Thanks, Rosemary, for the explanation. The changes look good to me then.